src/Faker/Provider/en_PH/Person.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Faker\Provider\en_PH;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}} {{suffix}}'
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+    );
+
+    protected static $firstNameMale = array(
+        'Anthony', 'Allan',
+        'Ben', 'Buboy', 'Bobby', 'Bentong', 'Bayani',
+        'Cydrick', 'Carlo', 'Cardo', 'Cesar',
+        'JC', 'Mark  Anthony', 'Jayfer', 'Paolo', 'Poy', 'Warren', 'Stephen', 'Maynard',
+        'Michael', 'Jeric', 'Rex', 'Benigno'
+    );
+
+    protected static $firstNameFemale = array(
```

Comment:
Please order this alphabetically. Lines should be 80 characters or less, this makes a diff easier to read, structure your code accordingly.